doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-deploying-on-ocp-kogito-cli.adoc

```diff
@@ -0,0 +1,81 @@
+[id='proc_kogito-deploying-on-ocp-kogito-cli']
+
+= Deploying {PRODUCT} services on {OPENSHIFT} using the {PRODUCT} CLI
```

Comment:
question, are we talking about kogito-cli or the oc tool?
shouldn't we also talk about the kogito cli as it does most of the hard work for the user?

Reply:
@spolti, Kogito CLI. Title says "... using Kogito CLI" and then the section is entirely about the Kogito CLI. I don't see the confusion.